pomodoro-cli.sh

```diff
@@ -17,16 +17,34 @@ function notify() {
 }
 
 ### Show a countdown timer and a message and updates without
-### cleaning the whole screen
-## Params: seconds {Number} - Countdown time in seconds 
+### cleaning the whole screen. Also kills apps if needed
+## Params: seconds {Number} - Countdown time in seconds
+## Params: is_focus_time {String} - "FOCUS" or "BREAK" or "LONG_BREAK"
+## Params: app_names {String} - Comma separated list of apps to kill
 ## Params: message {String} - Timer description
+## Item: IFS=',' is (Internal Field Separator) set to comma
 ## Returns: {String} - "hh:mm:ss - $message"
 function countdown(){
   secs=$1
+  is_focus_time=$2
+  app_names=$3
+  message=$4
+
+  IFS=','
+  read -ra arr <<< "$app_names"
+
   shift
```

Comment:
```suggestion
  read -ra arr <<< "$apps_to_kill"

  shift
  message=$*
```
I think it'll be easy if i add the message variable after shift

Reply:
yeah, no problem